build/sslstream-scenarios.yml

```
@@ -43,9 +43,9 @@ parameters:
     type: object
     default:
       - displayName: "1 connection"
-        arguments: --variable concurrency=1 --property concurrency=1
+        arguments: --variable connections=1 --property concurrency=1

```

Should we also change the property label?

```suggestion
        arguments: --variable connections=1 --property connections=1

```

---

I was going back and forth on this. On one hand, we already should have this table filled with some previous data, so renaming the column won't look pretty there... On the other hand, it would be nice to have QuicStream's result in the same table to be able to show them back to back in PowerBI. So I was thinking -- I will rename the table as well (to a common name) and it will also automatically solve the column problem :D